src/app/containers/ConsentBanner/index.amp.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import styled from 'styled-components';
+import Banner from './Banner/index.amp';
+
+const parentId = 'consent';
+const promptId = 'consent-prompt';
+const privacyId = 'privacy';
+const cookieId = 'cookie';
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eu: [
+      'at',
+      'be',
+      'bg',
+      'cy',
+      'cz',
+      'de',
+      'dk',
+      'ee',
+      'es',
+      'fi',
+      'fr',
+      'gb',
+      'gr',
+      'hr',
+      'hu',
+      'ie',
+      'is',
+      'it',
+      'li',
+      'lt',
+      'lu',
+      'lv',
+      'mt',
+      'nl',
+      'no',
+      'pl',
+      'pt',
+      'ro',
+      'se',
+      'si',
+      'sl',
+    ],
+  },
+};
+
+const ampConsentData = {
+  consents: {
+    'user-consent': {
+      promptIfUnknownForGeoGroup: 'eu',
+      promptUI: promptId,
+    },
+  },
+};
+
+const jsonInlinedScript = data => (
+  <script
+    type="application/json"
```

Comment:
Looks like if we're pulling in react-amphtml we can write this more cleanly: https://www.npmjs.com/package/react-amphtml#target-html

Reply:
That seems to only be for AmpState
```
<amp-state id="allAnimals">
  <script type="application/json">
    {
      "currentAnimal": "dog"
    }
  </script> 
</amp-state>
```
We need `amp-geo` not `amp-state` which I dont think they do